### PR TITLE
wording tweaks

### PR DESCRIPTION
**Pull request by gidsg**
Broke out these wording changes for the homepage into a separate PR.

@guardian/contentapi @guardian/content-platforms

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,10 +13,8 @@
    <body>
       <div class="landing">
          <h1><span class="gfont">g</span>Open Platform</h1>
-        
-        <p> Access The Guardian content. Entire content</p>
-        <p> Realtime updated and open to everyone.</p>
-        <p> What you will build with?</p>
+          <p>Integrate the Guardian's content into your website or app</p>
+          <p>What you will build with it?</p>
```
Comment (nicl) [inline index.html]: Should we just scrap this line? It's a bit cheesy I reckon.

**Comment by nicl** [comment]
In general :+1: apart from the note I added above.

File: index.html (hunk 1 above)
Comment (mchv) [inline index.html]: I would like to keep `Realtime updated and open to everyone.` and tend to prefer keeping 3 sentences (with decreasing sentences length) but open to change `What you will build with it?` with something less cheesy.
@nicl any ideas?

**Comment by gidsg** [comment]
@nicl wants fewer lines.
@mchv wants more.
I think this proves website-design-by-comittee does not work.
Let's talk about it at stand-up.

File: index.html (hunk 1 above)
Comment (rrees) [inline index.html]: "Please stop scraping our fucking site"? Or is that just me?

Comment (philwills) [inline index.html]: @rrees Perhaps, "Please fucking stop scraping our site"?  I like to think our site has a broader range of content than your wording implies.

Comment (rrees) [inline index.html]: I stand corrected

Comment (rrees) [inline index.html]: More srsly: "Open to all"

Comment (nicl) [inline index.html]: 'Open to all' works for me. So perhaps something like:

```
API access to The Guardian's journalism.
Updated in real-time. 
Open to all.
```

Comment (mchv) [inline index.html]: :+1:

**Comment by mchv** [comment]
I am fine with latest @nicl change, let's merge it :dancer: !

File: index.html (hunk 1 above)
Comment (nicl) [inline index.html]: p.s. @mchv just realised, but it keeps to your 3 sentences, of decreasing length rule too!

**Comment by gidsg** [comment]
@nicl Style guide says:

> lc for newspapers (the Guardian), magazines (the New Statesman), pubs (the Coach and Horses), bands (the Black Eyed Peas, the Not Sensibles, the The), nicknames (the Hulk, the Red Baron), and sports grounds (the Oval).

**Pull request merged**